src/Symfony/Bridge/Twig/Extension/IntlExtension.php

```diff
@@ -0,0 +1,92 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Twig\Extension;
+
+use Symfony\Component\Intl\Intl;
+
+/**
+ * Provides integration of the Intl component with Twig.
+ *
+ * @author Jules Pietri <jules@heahprod.com>
+ */
+class IntlExtension extends \Twig_Extension
+{
+    private $defaultLocale;
+
+    public function __construct($defaultLocale = null)
+    {
+        $this->defaultLocale = $defaultLocale;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function getFilters()
+    {
+        return array(
+            new \Twig_Filter('localized_country', array($this, 'getLocalizedCountry')),
+            new \Twig_Filter('localized_language', array($this, 'getLocalizedLanguage')),
+            new \Twig_Filter('localized_locale', array($this, 'getLocalizedLocale')),
+        );
+    }
+
+    /**
+     * @param string      $country
+     * @param string|null $displayLocale
+     *
+     * @return null|string
+     */
+    public function getLocalizedCountry($country, $displayLocale = null)
+    {
+        if (!is_scalar($country)) {
+            throw new \InvalidArgumentException(sprintf('Expected a scalar got "%s"', is_object($country) ? get_class($country) : gettype($country)));
+        }
```

Comment:
We're not doing such checks anywhere else in Symfony, so this needs to be removed.